index.html

```diff
@@ -207,7 +212,13 @@ <h3 class="project-heading">
                 </ul>
               </li>
               <li class="display-flex-width btn-3 btn-project-1">
-                <button class="action-button" type="button">See Project</button>
+                <button
+                  onclick="modalFunction(0)"
+                  class="action-button"
+                  type="button"
+                >
+                  See Project
+                </button>
```

Comment:
* Kindly adjust the `see live` and `see source` buttons, the contents should be properly centered too in the mobile popup window, this is how the ORIGINAL: looks
![image](https://user-images.githubusercontent.com/52497246/127350279-fa8b8b3b-b95e-4634-860f-664a2c95d0ea.png)
* YOUR design: 
![image](https://user-images.githubusercontent.com/52497246/127350603-a14aadfc-46c8-49f1-b9fc-22a1cf8017b2.png)